rewrite-test/src/main/kotlin/org/openrewrite/RecipeTest.kt

```diff
@@ -70,7 +70,7 @@ interface RecipeTest {
 
         assertThat(result).`as`("The recipe must make changes").isNotNull()
         assertThat(result!!.after).isNotNull()
-        assertThat(result.after!!.printTrimmed(treePrinter ?: TreePrinter.identity<Any>()))
+        assertThat(result.after!!.print(treePrinter ?: TreePrinter.identity<Any>()))
```

Comment:
Causes a bunch of rewrite-java test failures, but I think those are actually problems in rewrite-java recipes. Whitespace is relevant, so tests shouldn't use a result tree print result modified by StringUtils.trimIndent, which is done in Tree.printTrimmed but not in Tree.print.